test/msd_test.py

```diff
@@ -296,6 +296,11 @@ def test_mass_storage(workspace, parent_test):
     test.set_flush_size(0x1000)
     test.run()
 
+    if target.name == "BlueNinja":
+        test_info.info("Skip the remaining MSD test.")
+        test_info.info("Writing incomplete binary to TZ10xx MCU, CoreSight becomes unresponsive.")
```

Comment:
MCU manufacturer told me that it is not specified in the document.

The they say:

> When the vector table to write incomplete program is as follows, it is impossible to negotiate CoreSight and debug adapter.
> 
> 1. HardwareFault Exception has occurred
> 2. unrecoverable Exception has occurred
> 3. SystemReset
> 4. Go back to 1
> 
> Restoration is Flash chip erase started with a "SPI Flash erase mode" the MCU.

BlueNinja there is a way to start the hardware to "SPI Flash erase mode".
But, can not be controlled from the interface.